wp-config-pantheon.php

```diff
@@ -59,8 +59,14 @@
         $scheme = 'https';
         $_SERVER['HTTPS'] = 'on';
     }
-    define('WP_HOME', $scheme . '://' . $_SERVER['HTTP_HOST']);
-    define('WP_SITEURL', $scheme . '://' . $_SERVER['HTTP_HOST']);
+	
+    if ( ! defined( 'WP_HOME' ) ) {
```

Comment:
Thanks for the PR. Not a blocker, but I prefer the "defined or define" syntax, e.g.:

```php
defined( 'WP_HOME' ) or define('WP_HOME', $scheme . '://' . $_SERVER['HTTP_HOST']);
defined( 'WP_SITEURL' ) or define('WP_SITEURL', $scheme . '://' . $_SERVER['HTTP_HOST']);
```

Reply:
Thank you for the feedback. Applied.